README.md

```
@@ -3,6 +3,19 @@
 NestJS + TypeORM + PostgreSQL 기반 Posts CRUD API.
 **Repository Pattern**으로 데이터 액세스를 추상화하고, **CQRS Pattern**으로 읽기/쓰기 관심사를 분리한다.
 
+### 주요 기능
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)